cms/templates/certificate_page.html

```diff
@@ -0,0 +1,134 @@
+{% extends "base.html" %}
+{% load static wagtailimages_tags image_version_url %}
+
+{% block title %}{{ site_name }} | Certificate for: {{ page.product_name }}{% endblock %}
+
+{% block seohead %}
+    {{ block.super }}
+    <meta property="og:site_name" content="{{ site_name }}" />
+    <meta property="og:type" content="website" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@MITxonedX" />
+    <meta name="twitter:image" content="{{ share_image_url }}" />
+    <meta property="og:title" content="{{ site_name }} | Certificate for: {{ page.product_name }}" />
+    <meta property="og:description" content="Certificate for {{ page.product_name }} awarded by {{ site_name }}." />
+    <meta property="og:url" content="{{ request.build_absolute_uri }}" />
+    <meta property="og:image:url" content="{{ share_image_url }}" />
+    <meta property="og:image:width" content="{{ share_image_width }}" />
+    <meta property="og:image:height" content="{{ share_image_height }}" />
+    <meta property="og:image:alt" content="A certificate from MIT xPRO" />
```

Comment:
Currently, this template will not work as we haven't passed any context to this template as well as this template needs tweaks according to bootcamps.